### PR TITLE
Blog kubecon eu 22

### PR DESCRIPTION
**Pull request by aburdenthehand**
Signed-off-by: aburdenthehand <aburden@redhat.com>

Belated blog post about the two events that KubeVirt presented at KubeCon EU 22.
Unfortunately we only have a recording link for one of them. 

**Special notes for your reviewer**:
I haven't been able to build this locally to test (see #850) so hopefully this PR will provide a netlify build to check. 
Have run spellcheck.
@alicefr - can you please have a look to see that I have accurately presented your talk?

### CODE REVIEW

File: _posts/2022-06-28-KubeVirt-at-KubeCon-EU-2022.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+This was an opportunity for KubeCon attendees (virtual and physical) to ask questions and discuss any topics, and our presenters covered the following: Introduction to KubeVirt, live migration, Istio integration, and CDI hotplug/resize.
+Despite some initial technical issues and improvised changes, this session went really well. We had about ~25 consistent attendees, and we received a good range of Q&A and interaction with the attendees on all topics presented. It was a very solid 45 minutes.
+Unfortunately, due to a miscommunication, there is no recording of this session. 
+
+A huge thanks to the presenters for their time and collaboration in preparing for this. 
+
+## Maintainer Track
+
+Later that day, on the Maintainer Track, Alice also gave an in-depth breakdown of a whole slew of new KubeVirt features and showed a demo with the KubeVirt Cluster API: deploying Kubernetes on top of Kubernetes.
+You can watch [the CNCF recording here](https://youtu.be/L9H0pz5PpKo).
```
Comment (alicefr) [inline _posts/2022-06-28-KubeVirt-at-KubeCon-EU-2022.md:40]: Maybe also the link to the session can be useful as it has the slides and the demo recording: https://sched.co/ytu1

Comment (aburdenthehand) [reply _posts/2022-06-28-KubeVirt-at-KubeCon-EU-2022.md]: Great idea. Done. Looks like Sched links are fairly permanent too, although I wonder if we shouldn't create a KubeCon EU 22 folder in the community repo and store them there as well.

**Comment by aburdenthehand** [comment]
@cwilkers and @mazzystr 
Hello hello - any chance this could get a review? Thank you!

**Comment by cwilkers** [comment]
I was able to update things until I could see the test render locally. I'll do PRs there in the issue you raised (thanks!) and in the meantime, all the formatting looks great here, so:

/approve
/lgtm

**Pull request merged**